### PR TITLE
Feature/add statement#5667

### PR DESCRIPTION
**Pull request by HavierD**

### CODE REVIEW

File: indexer/src/main/resources/config_files/portal_records_index_schema.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
           }
         }
       },
+      "sci:citation": { "type": "text" },
```
Comment (utas-raymondng) [inline indexer/src/main/resources/config_files/portal_records_index_schema.json:81]: Is this schema factor back into the ogcapi test?

**Review by utas-raymondng — CHANGES_REQUESTED** [review]
Minor comments

File: indexer/src/main/resources/config_files/portal_records_index_schema.json (hunk 1 above)
Comment (HavierD) [reply indexer/src/main/resources/config_files/portal_records_index_schema.json:81]: Thank you! The pr of ogcapi is here: https://github.com/aodn/ogcapi-java/pull/67 
Test is running. Please feel free to review it

**Review by utas-raymondng — APPROVED** [review]
LGTM

**Pull request merged**